app/views/results.nunjucks

```diff
@@ -26,20 +26,7 @@
 
     <a href="{{ SITE_ROOT }}" class="link-back">Back</a>
     <h1 class="sr-only" data-results="{{gps.length}}">Search results</h1>
-    <h2 class="results__header">GP {% if gps.length === 1 %}surgery{% else %}surgeries{%endif %} matching '{{ search }}' </h2>
```

Comment:
Question for @mcgrathjoe / interaction designers / looking at the prototype: 
- what do we do with no results via the JS solution
- what do we do with an empty search string via the JS solution
- how do we test the js solution